lec15-data-viz.Rmd

```diff
@@ -1,4 +1,241 @@
+
 ---
-title: "Data visualization in R"
+title: Data visualization in R
+author: Lindsay Coome
 ---
 
+## Lesson preamble
+
+> ### Learning Objectives
+>
+> - Making better plots for outreach and communication.
+> - Review how to produce scatter plots with trendlines using ggplot.
+> - Make bar and other graphs using ggplot.
+> - Advanced faceting and segmenting data.
+> - Altering asthetic features in ggplot.
+> - Making interactive and quick graphs.
+> 
+> ### Lesson outline
+>
+> - How to visualize data (20 min)
+> - ggplot basics (10 min)
+> - Basics and faceting (15 min)
+> - Aesthetics and plot types (20 min)
+> - Interactive and quick graphing (10 min)
+
+-----
+
+#WHAT MAKES A GOOD GRAPH?
+> #### Challenge
+>
+> What makes a bad graph? What makes an ugly graph to you?
+>
+
+
+There are many considerations to take into account when creating a graph.
+Data types:
+* Nominal
+* Ordinal
+* Interval
+* Ratio
+And data features:
+* Scalar
+* Vector
+* Tensor
+
+Ask: "what kind of data/information is it?"
+* Numerical?
+* What type?
+* Spatial?
+* How many dimensions?
+* Nominal/categorical?
+* Narrative?
+
+## TUFTE'S GUIDELINES
+* Reduce non-data ink
+* Enhance the data ink
+
+Reduces the proportion of graphic’s ink devoted to the non-redundant display of data-information
+
+Avoid "chartjunk" - extraneous visual elements that detract from message
+
+Visual Attribute    | Non-Data | Data
+--------------------|----------|-------
+Line Thickness      | Thin     | Thick
+Size                | Small    | Big
+Brightness Contrast | Decrease | Increase
+Colour Saturation   | Decrease | Increase
+
+#COLOUR RULES
+* Large background colors should be quiet, muted to let brighter colors stand out
```

Comment:
I didn't quite follow the conversation about color/colour from before, but maybe keep consistent (a few lines below, you go back to 'colo*u*r'